attributes/nsqd.rb

```diff
@@ -83,8 +83,8 @@
 # -statsd-address='': UDP <addr>:<port> of a statsd daemon for writing stats
 default['nsq']['nsqd']['statsd_address'] = '127.0.0.1:8125'
 
-# -statsd-interval=30: seconds between pushing to statsd
-default['nsq']['nsqd']['statsd_interval'] = '30'
+# -statsd-interval=30s: seconds between pushing to statsd
+default['nsq']['nsqd']['statsd_interval'] = '30s'
```

Comment:
is this backward compatible to the 0 versions?

Reply:
Yeah, I tested with 0.3.8.